networkmanager_dmenu

```diff
@@ -787,15 +797,18 @@ def run():
         gsm_actions = []
         wwan_actions = []
 
-    saved_actions = None
     list_saved = CONF.getboolean('dmenu', 'list_saved', fallback=False)
-    saved_actions = []
```

Comment:
I removed this because `saved_actions` now gets set either way anyways, and the `saved_actions = None` above was unneeded/overwritten even before my change - thus I removed it as well.